docs/configuring-playbook-dimension.md

```diff
@@ -0,0 +1,48 @@
+# Setting up Dimension (optional)
+
+Dimension integrations manager installation is disabled by default. You can enable it in your configuration file (`inventory/matrix.<your-domain>/vars.yml`):
```

Comment:
Maybe "Dimension integrations manager" could link to https://dimension.t2bot.io/, so that people who don't know what it is, can find out.